packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart.ts

```diff
@@ -76,6 +76,9 @@ export class ForceFunctionStartExecutor extends SfdxCommandletExecutor<string> {
   }
 
   public execute(response: ContinueResponse<string>) {
+    const runtimeDetection: RegExp = new RegExp(
```

Comment:
nit: upper case and move out of the class / make static since this is a constant

Reply:
also in the variable name describe we are detecting debug type here